### ISSUE TITLE
DISCUSSION: Make game less open?

### ISSUE THREAD

**Issue by Miepee** [randovania/YAMS#231]
AM2R is notoriously *open*, which works fine for single-player randos, but for Multiworlds can become a real pain. It's not uncommon for am2r players to finish their game considerably earlier than everyone else and then be stuck in BK at the end for their final clean-up, even with shuffled wall jumps and IBJ.

An idea that came to mind was to bring back the lava. This would mean that you need to collect X DNA to lower the lava per area, which would slow down the am2r gameplay by a bit. (And technically also bring it a bit closer to vanilla)

Big issues that I see:
- It now suddenly always requires 46 DNA again, which, if theres still the limitation on DNA on metroids only, would mean metroids become absolutely useless again. 
- Starter preset AM2R would **always** have to do surface alpha
- Random start becomes significantly more likelier to either fail or give you screw attack, as the main caves would be full of lava/blocks and be thus intraversible.
- there's a lot less thinking about logic involved, and it basically goes back to "just linearly clear the the game from top to bottom, do some occasional backtracking"

---

Could potentially toy around with the idea of having only *one* lava check, somewhere in the middle of the game? Which could then have a configurable amount of how much DNA it needs in order to pass. 

---

Another idea was also to move the Mines Super crystal into western cave instead, so that you need supers (or screw) to get into the lower areas (or upper areas if you start somewhere at the bottom)

---

#230 will also probably contribute to making it less open, considering it would lock an entire area behind one item thats otherwise free

**Comment by AbyssalCreature** [comment]
Another idea clearly should be adding Geothermal Escape as an option :)

**Comment by Miepee** [comment]
Geothermal doesnt really help solve the problem.

**Comment by AbyssalCreature** [comment]
Yeah that's true, anyways regarding the actual topic of this issue:

- If the lava really wants to be done, it could be a great time to add a few items in the area before the first lava drop (right after the first alpha dies), I'm pretty sure I mentioned this a few times but I'll post the images here. They were meant to be extra checks that you can get here so first alpha didn't feel really punishing to check early, but with the inclusion of this new thing it can open more possibilities.
![image](https://github.com/user-attachments/assets/0cd56036-2fc1-49f9-b231-1fcb093e39c0)
![image](https://github.com/user-attachments/assets/4597aae4-d407-4eea-abd1-cdc897a3418c)
![image](https://github.com/user-attachments/assets/720965ee-50fd-420a-bd4a-ea8a74783911)
![image](https://github.com/user-attachments/assets/300b2f10-4c7b-4edf-805c-3eac8203c943)

- Moving Mines Crystal feels out of place IMO, but whatever needs to be done to make it less open.
- If the middle lava check is done, it could be the one that cuts the way to The Tower outside of the Octroll lava pond.

**Comment by Miepee** [comment]
> it could be a great time to add a few items in the area before the first lava drop

If that's done, the positioning is definitely gonna be different than in your image. Because your item placement causes all items to be just "on the way". They also dont help with the DNA problem.

 > Moving Mines Crystal feels out of place IMO
 
 Eh, its really just "move the crystal from the current subarea to the next subarea which just happens to gate the main path". The environment for both is already similar, so it wouldnt really feel too out of place for me at least.
 
 > If the middle lava check is done, it could be the one that cuts the way to The Tower outside of the Octroll lava pond.
 
 That was what i meant with it, yes.

**Comment by JeffGainsNGames** [comment]
to go along with #230 could also add heat to rooms with lava
a3 lower interior
path to research site
mines metroids and access
a6
probably would just mean folks heat run though since they are smaller in size than an entire area (4)

**Comment by Miepee** [comment]
most things would be easily ran through yeah.
A3 lower i can imagine needing either a ton more HP tho since heat + lava + time to kill gamma adds up (itd also make that area even worse than it already is...)
path to research site would just mean people dont damage tank through the lava
mines can probably be just tanked
and a6 most of the time can be easily skipped if you have screw anyway.
